src/views/Home.vue

```diff
@@ -1,68 +1,132 @@
 <template>
   <ion-page>
-    <ion-header :translucent="true">
-      <ion-toolbar>
-        <ion-title>Blank</ion-title>
-      </ion-toolbar>
-    </ion-header>
-    
+  <ion-toolbar>
+    <ion-button id="signInButton" button @click="buttonClickSignIn()">{{user.firstName}}</ion-button>
+    <h1>LOGO</h1>
```

Comment:
If this is an image, use an image tag and just use a placeholder for that image for now.

Reply:
done